packages/core/upload/admin/src/components/AssetDialog/index.js

```diff
@@ -98,8 +98,8 @@ export const AssetDialog = ({
   if (isLoading) {
     return (
       <ModalLayout onClose={onClose} labelledBy="asset-dialog-title" aria-busy>
-        <DialogTitle />
-        <Flex justifyContent="center" paddingTop={4} paddingBottom={4}>
+        <DialogHeader />
+        <Flex justifyContent="center" paddingTop={4} paddingBottom={4} minHeight={pxToRem(560)}>
```

Comment:
How did you come up with this number? :)

Reply:
I was searching for a better alternative 😅
the issue is that we have the ModalBody that is fixed to `max-height 60vh` in the DS
but we are not wrapping the Tabs inside the ModalBody so the best I found is `min-height: calc(60vh + 80px);`
I wonder why Tabs is not wrapped in ModalBody that would be easier, I'll try it, WDYT?

Reply:
well wrapping Tabs in ModalBody causes issue, so I still need to account for the height of the Tabs + button with `min-height: calc(60vh + 80px);` 🤔

Reply:
Replied below. Makes sense to me :)